netci.groovy

```diff
@@ -638,6 +638,10 @@ combinedScenarios.each { scenario ->
                                         if (Constants.jitStressModeScenarios.containsKey(scenario)) {
                                             if (enableCorefxTesting) {
                                                 // Sync to corefx repo
+                                                // Move coreclr files to a subdirectory, %workspace%/clr. Otherwise, corefx build 
+                                                // thinks that %workspace% is the project base directory.
+                                                buildCommands += "powershell new-item clr -type directory -force"
+                                                buildCommands += "powershell foreach (\$x in get-childitem -force) { if (\$x.name -ne \'clr\') { move-item \$x clr }}"
```

Comment:
You can use a single quoted string here to avoid the escaped powershell variables  Btw are you sure this works?  I thought you had to do: powershell -Command "& <command>"